### DISCUSSION TITLE
Is on-board storage possible / could it be added?

### DISCUSSION THREAD

**Discussion by jerabaul29** [pine64/OpenPineBuds#105] [Q&A] [answered]
It would be very nice to be able to upload audio files to the PineBud Pro, program what order to play them in, and then be able to run or whatever with just the PineBud Pro, without phone.

Is it something that would be possible / could be added? Or would this be impossible given the current hardware? If so, maybe some option for a future next version? :)

**Comment by shymega** [comment, marked as answer]
Moving to Q&A.

No, not possible with this hardware, and I would personally be against the overhead.
